src/doc/rustc-dev-guide/src/SUMMARY.md

```diff
@@ -75,6 +75,7 @@
 - [Prologue](./building/bootstrapping/intro.md)
 - [What Bootstrapping does](./building/bootstrapping/what-bootstrapping-does.md)
 - [How Bootstrap does it](./building/bootstrapping/how-bootstrap-does-it.md)
+- [`tracing` in bootstrap](./building/bootstrapping/bootstrap-tracing.md)
```

Comment:
Can you make it "debugging in bootstrap" instead? You can leave a FIXME note that says something like "extend this page". Having a page that explains ways of debugging bootstrap would be great I think.

Reply:
Changed to "debugging bootstrap" and left a FIXME.